decidim/decidim
# Enable admins answer to closed comments

#### :tophat: What? Why?
This PR enables administrators to answer comments or post new ones even after they are closed to participants.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #?
- Fixes #12358

#### Testing

1. Log in as administrator.
2. Enter the admin panel and navigate to a process that have a proposals/debates component with comments enabled.
3. Block comments in that component in the steps settings.
4. Navigate to a proposal/debates on the frontend where comments have been blocked.
5. See that no comments can be submitted or reply unless you're an administrator and there is a warning message indicating that comments have been disabled

### :camera: Screenshots
![image](https://github.com/decidim/decidim/assets/116598037/235c1b1a-2de7-43a0-b5df-e0e9488517c0)
warning message for blocked comments

:hearts: Thank you!
